### ISSUE TITLE
NTR: starch intolerance

### ISSUE THREAD

**Issue by cmungall** [obophenotype/human-phenotype-ontology#4953]
**Preferred term label:**

starch intolerance

**Synonyms**

amylose-amylopectin intolerance

**Definition (free text, please give PubMed ID)**

 An inability to digest starch

**Parent term (use hpo.jax.org/app)**

obo:HP_0002024 ! Malabsorption

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

children of

https://www.ebi.ac.uk/ols/ontologies/ordo/terms?iri=http%3A%2F%2Fwww.orpha.net%2FORDO%2FOrphanet_35122

See also https://github.com/monarch-initiative/mondo/issues/886

**Your nano-attribution (ORCID)**

0000-0002-6601-2165
